### PR TITLE
Update type hints of `core.signing`

### PR DESCRIPTION
**Pull request by Viicos**
# I have made things!

### CODE REVIEW

File: django-stubs/core/signing.pyi [hunk 1]
```diff
@@ -20,31 +25,35 @@ class JSONSerializer:
...
-    key: str
+    key: bytes | str
+    fallback_keys: list[bytes | str]
     sep: str
-    salt: str
+    salt: bytes | str
     algorithm: str
     def __init__(
         self,
+        *args: Unpack[tuple[()]],
```
Comment (Viicos) [inline django-stubs/core/signing.pyi]: This to make stubtest happy.

Two other solutions:
- Add the `__init__` in the ignorelist
- Add an overload, one without `args` (and a empty stararg as it is currently on django's `main`), one with `*args: Any` and `typing_extensions.deprecated` applied to this overload

Comment (sobolevn) [reply django-stubs/core/signing.pyi]: Let's just use `Any`

**Review by sobolevn — APPROVED** [review]
Thanks! Can you please try `deprecated` here?
Just as an experiment.

**Comment by Viicos** [comment]
This is how it looks with Pyright and a LSP implementation:
image

**Review by sobolevn — APPROVED** [review]
This is cool! Thanks a lot.

**Pull request merged**